app/views/good_job/shared/_filter.erb

```diff
@@ -71,5 +82,29 @@
         });
       })
     })
+
+    document.addEventListener("DOMContentLoaded", () => {
```

Comment:
I don't write a lot of javascript, so I wouldn't be surprised if there's a better way to do this